Formula/hubble.rb

```diff
@@ -0,0 +1,22 @@
+require "open3"
+
+class Hubble < Formula
+  desc "Network, Service & Security Observability for Kubernetes using eBPF"
+  homepage "https://github.com/cilium/hubble"
+  url "https://github.com/cilium/hubble/archive/refs/tags/v0.8.1.tar.gz"
+  sha256 "ebb54783c57e7997986dcc5a73f4f845015920b857dd348abd7a02e5d3af65ec"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "hubble"
+    bin.install "hubble"
```

Comment:
Based on the makefile, this does:
```
$(GO_BUILD) $(if $(GO_TAGS),-tags $(GO_TAGS)) -ldflags "-w -s -X 'github.com/cilium/hubble/pkg.GitBranch=${GIT_BRANCH}' -X 'github.com/cilium/hubble/pkg.GitHash=$(GIT_HASH)' -X 'github.com/cilium/hubble/pkg.Version=${VERSION}'" -o $(TARGET)
```
We could just replace this with:
```ruby
  ldflags = "-s -w -X github.com/cilium/hubble/pkg.Version=#{version}"
  system "go", "build", *std_go_args(ldflags: ldflags)
```
(It seems like the other flags aren't necessary, and they would be "blank" anyway when compiling from the tarball – based on your recent PR upstream.)